connor/database/queries.go

```diff
@@ -0,0 +1,70 @@
+package database
+
+const (
+	insertOrders = "INSERT INTO ORDERS(ID, PRICE, HASHRATE, START_TIME, BUTTERFLY_EFFECT, ACTUAL_STEP) VALUES (:ID, :PRICE, :HASHRATE, :START_TIME, :BUTTERFLY_EFFECT, :ACTUAL_STEP)"
+	insertDeals  = "INSERT INTO DEALS(ID, STATUS, PRICE, ASK_ID, BID_ID, DEPLOY_STATUS, START_TIME, LIFETIME ) VALUES (:ID, :STATUS, :PRICE, :ASK_ID, :BID_ID, :DEPLOY_STATUS, :START_TIME, :LIFETIME )"
+	insertToken  = "INSERT INTO TOKENS (TOKEN_ID, NAME, USD_PRICE, NET_HASHES_SEC, BLOCK_TIME, BLOCK_REWARD,  PROFIT_PER_MONTH_USD, DATE_TIME) VALUES (:TOKEN_ID, :NAME, :USD_PRICE, :NET_HASHES_SEC, :BLOCK_TIME, :BLOCK_REWARD, :PROFIT_PER_MONTH_USD, :DATE_TIME)"
+	insertPools  = "INSERT INTO POOLS (POOL_ID, POOL_BALANCE, POOL_HASH, W_ID, W_REP_HASH,W_AVG_HASH, BAD_GUY, ITERATIONS, TIME_START, TIME_UPDATE ) VALUES (:POOL_ID, :POOL_BALANCE, :POOL_HASH,  :W_ID, :W_REP_HASH, :W_AVG_HASH, :BAD_GUY, :ITERATIONS, :TIME_START, :TIME_UPDATE)"
+
+	updateOrders           = "UPDATE ORDERS SET BUTTERFLY_EFFECT = ?	 WHERE ID = ?"
+	updateDeals            = "UPDATE DEALS SET DEPLOY_STATUS = ?	 WHERE ID = ?"
+	updateStatusPoolDB     = "UPDATE POOLS SET BAD_GUY = ? WHERE W_ID = ?"
+	updateReportedHashrate = "UPDATE POOLS SET W_REP_HASH = ? WHERE W_ID = ?"
+	updateAvgPool          = "UPDATE POOLS SET W_AVG_HASH = ? WHERE W_ID = ?"
+	updateIterationPool    = "UPDATE POOLS SET ITERATIONS = ? WHERE W_ID = ?"
+
+	getCountFromDb    = "SELECT COUNT(ID) AS count FROM ORDERS"
+	getLastActualStep = "SELECT MAX(ACTUAL_STEP) AS max FROM ORDERS WHERE BUTTERFLY_EFFECT = 2"
+
+	getOrders          = "SELECT * FROM ORDERS"
+	getDeals           = "SELECT * FROM DEALS"
+	getWorkersFromPool = "SELECT * FROM POOLS"
+
+	deals = `
```

Comment:
names of queries which create tables are meaningless, better to name it smth like a `recreateDealsTable`